README.md

```diff
@@ -180,11 +180,11 @@ dependencies {
 }
 ```
 
->We also provide `kotlinx-serialization-core` artifact that contains all serialization API but does not have bundled serialization format with it
+>We also provide `kotlinx-serialization-core` artifacts that contains all serialization API but does not have bundled serialization format with it
```

Comment:
While technically, several core artifacts exist (for jvm, native, etc...), ideologically, they are all the same. Moreover, platform artifacts have different coordinates (`kotlinx-serialization-core-jvm`, etc). From a user standpoint, only one `kotlinx-serialization-core` should be used. Therefore, I'd prefer to leave this word in a singular form.

Also, you've had to change `contains` to match `artifacts`.

Reply:
ok as your wish